demo/scripts/controls/ribbonButtons/contentModel/tableBorderColorButton.ts

```diff
@@ -0,0 +1,29 @@
+import { getButtons, getTextColorValue, KnownRibbonButtonKey } from 'roosterjs-react';
+import { isContentModelEditor } from 'roosterjs-content-model-editor';
+import { RibbonButton } from 'roosterjs-react';
+
+const originalButton = getButtons([KnownRibbonButtonKey.TextColor])[0] as RibbonButton<
+    'buttonNameTableBorderColor'
+>;
+
+/**
+ * @internal
+ * "Table Border Color" button on the format ribbon
+ */
+export const tableBorderColorButton: RibbonButton<'buttonNameTableBorderColor'> = {
+    ...originalButton,
+    unlocalizedText: 'Table Border Color',
+    iconName: 'ColorSolid',
+    isDisabled: formatState => !formatState.isInTable,
+    onClick: (editor, key) => {
+        // This check will always be true, add it here just to satisfy compiler
+        if (key != 'buttonNameTableBorderColor' && isContentModelEditor(editor)) {
+            editor.setTableBorderColor(
```

Comment:
What is this function for? Is it to change border color of current selected table (if any) or just set different table border color?

Reply:
Just set different border color. The function that applies changes is named applyBorderFormat.

Reply:
Can you elebrate more about the funcationality of this function?

Assume I have two tables, and one is selected. Now I call this function to set to a different border color, then what is the expected result?

Reply:
Internally the editor core will change the border color that will be applied by applyBorderFormat if called. No changes to any table in the editor.

Reply:
Then if I call applyBorderFormat, will both table be formatted, or just one?

Reply:
Only what is selected will be formatted.

Reply:
In that case, why not just format the table directly, and no need to store the format inside editor.

Reply:
Because that is the requirement of the function. There needs to be an editor-wide format, independent of any table, that only gets applied though the Apply Borders functions, currently only All Borders is implemented.